packages/code-du-travail-modeles/src/modeles/conventions/44_industries_chimiques/__tests__/rupture-conventionnelle/calculate.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { RuptureConventionnellePublicodes } from "../../../../../publicodes";
+import { CategoryPro44 } from "../../salary";
+
+const engine = new RuptureConventionnellePublicodes(
+  modelsRuptureConventionnel,
+  "44"
+);
+
+describe("Gestion des licenciements pour la CC 44", () => {
+  describe("Missing variables", () => {
+    test("Sans arguments", () => {
+      const input = {
+        "contrat salarié . convention collective": "'IDCC0044'",
+      };
+
+      const result = engine.calculate(
+        input,
+        "contrat salarié . indemnité de licenciement . résultat conventionnel"
+      );
+      const missingArgs = result.missingArgs.filter(
+        (item) => item.rawNode.cdtn
+      );
+      expect(missingArgs).toHaveLength(2); //TODO: essayer de gérer les missingArgs qui utilisent un namespace présent dans après le dismissal-reason
+      expect(missingArgs[0].name).toBe(
+        "contrat salarié - convention collective - industries chimiques - indemnité de licenciement - catégorie professionnelle"
+      );
```

Comment:
```suggestion
      const { missingArgs } = engine.calculate(
        input,
        "contrat salarié . indemnité de licenciement . résultat conventionnel"
      );
      expect(missingArgs).toHaveNextMissingRule("contrat salarié . convention collective . industries chimiques . indemnité de licenciement . catégorie professionnelle");
```

Et ça retire le todo :)